sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewTestSuite.scala

```diff
@@ -530,4 +530,23 @@ class PersistedViewTestSuite extends SQLViewTestSuite with SharedSparkSession {
       spark.sessionState.conf.clear()
     }
   }
+
+  test("SPARK-37266: Optimize the analysis for view text of persistent view and" +
+    " fix security vulnerabilities caused by sql tampering") {
```

Comment:
we can shorten the test name: `SPARK-37266: view text can only be SELECT queries`